tests/test_kwargs_and_defaults.py

```diff
@@ -382,7 +382,7 @@ def test_args_refcount():
     arguments"""
     refcount = m.arg_refcount_h
 
-    myval = 54321
+    myval = object()
```

Comment:
"Changes an existing test" stood out to me when glancing through quickly (I can look carefully only later).

Why does this test (and test_pytypes.py) need to be changed?

Reply:
The free-threaded build immortalizes most constants, including numbers and byte literals, so they don't work well for refcount tests.

Here, `object()` is a good stand in for something that's refcounted. In `test_pytypes.py`, passing the literal through `str.encode` gives a non immortalized `bytes` object.

Reply:
I can pull out this change, and also change the check afterwards which includes the immortal object value. This is a more interesting check with `object()`, since you can't track increment and decrement if something is immortal.

Reply:
Done in #5150.